week-3/todo-list/style.css

```diff
@@ -1 +1,140 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 12px;
+  margin: 17px;
+  background-color: #fff;
+  color: black;
+  font-size: x-large;
+}
+.container-title {
+}
+body {
+  /* width: 100%;
+  min-width: 100vh;
+  background: linear-gradient(135deg, #153677, #4e085f); */
+  background-image: url(https://chrisowen101.github.io/ToDoListSolution/assets/world.jpg);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center;
+  background-size: auto 100%;
+  /* padding: 10px; */
+}
+body img {
+  width: 30%;
+  height: 30%;
+}
+.container {
+  margin: 0rem 0rem 0rem 26rem;
+}
+.todo-app {
+  width: 100%;
+  max-width: 699px;
+  background: #fff;
+  margin: 100px auto 20px;
+  padding: 6px 4px 60px;
+  border-radius: 6px;
+}
+.todo-app h2 {
+  padding: 0.5rem 1rem;
+  margin-bottom: 0;
+  background-color: rgba(0, 0, 0, 0.03);
+  border-bottom: 2px solid rgba(0, 0, 0, 0.125);
+  font-size: 17px;
+  font-weight: 100;
+}
+.row {
+  display: flex;
+  align-items: center;
+  /* justify-content: space-between; */
+  gap: 4px;
+  margin: 12px;
+  /* background: #edeef0; */
+  /* border-radius: 30px;
+  padding-left: 20px;
+  margin-bottom: 25px; */
+}
 
+input {
+  padding: 12px;
+  width: 50%;
+  border-radius: 3px;
+  border: solid 0.3px gray;
+  /* flex: 1;
+  border: none;
+  outline: none; */
+  /* background: transparent; */
+  /* margin: 12px; */
+}
+.add-btn {
```

Comment:
The add todo and remove all button have very similar styles, how can you reduce duplicate styling rules?

Reply:
thanks for your feedback dear nirmeet  , i will try my best to defector my code , if not i will keep it in mind to use in my other projects (: